public/data/data.json

```
@@ -1 +1,110 @@
-[{ "data": 1 }]
+[
+  {
+    "id": 1,
+    "nickName": "dayeon",
+    "profileImageUrl": "https://kream-phinf.pstatic.net/MjAyMjA3MThfMzUg/MDAxNjU4MTIxODA5NTg3._KJZLabb2vblzxRDQehpHOYRu26ZbloFrSB2vX-LrAwg.PW5LTmbpxZ3T-b58m19DN6mhNZM8JR5gp2xTPhvdrPEg.JPEG/p_b93beee2fbab46b781f6938a61a2fad5.jpg?type=sl",
+    "picUrl": "https://kream-phinf.pstatic.net/MjAyMjEyMjdfMjUg/MDAxNjcyMTE3NDk5NjUy.3SPK47x9T-6_Ls2eVwIuZlmNS6dz8u9tpSFBQ64zGa8g.ZE9HL47jVhxZSol6QKReMEwEmK7bepPotcvTdH7N0Ygg.JPEG/p_9014f3561b0c45518adcf097cc1ad4a4.jpeg?type=m",
```

보통 이런 key값은 thumbnail, image 로 가져가는 편입니다.
```suggestion
    "thumbnail": "https://kream-phinf.pstatic.net/MjAyMjA3MThfMzUg/MDAxNjU4MTIxODA5NTg3._KJZLabb2vblzxRDQehpHOYRu26ZbloFrSB2vX-LrAwg.PW5LTmbpxZ3T-b58m19DN6mhNZM8JR5gp2xTPhvdrPEg.JPEG/p_b93beee2fbab46b781f6938a61a2fad5.jpg?type=sl",
    "image": "https://kream-phinf.pstatic.net/MjAyMjEyMjdfMjUg/MDAxNjcyMTE3NDk5NjUy.3SPK47x9T-6_Ls2eVwIuZlmNS6dz8u9tpSFBQ64zGa8g.ZE9HL47jVhxZSol6QKReMEwEmK7bepPotcvTdH7N0Ygg.JPEG/p_9014f3561b0c45518adcf097cc1ad4a4.jpeg?type=m",
```

---

하지만 이미 backend와 키값을 다 맞추셨다면 바꾸지 않고 진행하는 편이 좋아보여요!